khmer/__init__.py

```diff
@@ -359,6 +363,17 @@ def __new__(cls, k, starting_size, n_tables):
         return counttable
 
 
+class QFCounttable(_QFCounttable):
```

Comment:
Note that now (in the dawn of the Glorious Revolution), we'll be removing this pattern -- this `__new__` method should just be implemented in the `__cinit__`.